pkg/restore/prioritize_group_version.go

```diff
@@ -0,0 +1,369 @@
+/*
+Copyright 2020 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package restore
+
+import (
+	"context"
+	"sort"
+	"strings"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/version"
+
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	"github.com/vmware-tanzu/velero/pkg/archive"
+	"github.com/vmware-tanzu/velero/pkg/client"
+	"github.com/vmware-tanzu/velero/pkg/features"
+)
+
+// meetsAPIGVRestoreReqs determines if the feature flag has been
+// enabled and the backup objects use Status.FormatVersion 1.1.0.
+func (ctx *restoreContext) meetsAPIGVRestoreReqs() bool {
+	return features.IsEnabled(velerov1api.APIGroupVersionsFeatureFlag) &&
+		ctx.backup.Status.FormatVersion == "1.1.0"
```

Comment:
Should we be strictly equal, or >=?

Reply:
I had no idea Go was able to sort versions like this in strings the way I expected. Pretty neat. Yes, because Go can do that, `>=` works fantastic.